### ISSUE TITLE
Evaluation of resource 

### ISSUE THREAD

**Issue by leoalenc** [LFG-PTBR/MorphoBr#7]
Os diferentes formatos de entrada do dicionário original se classificam em um número limitado de tipos. Podemos avaliar a conversão  para o formato atual, realizada pelo módulo em linguagem piton, testando o módulo nesses casos específicos.  desse modo, vou preparar essa bateria de testes e submeter ao resto da equipe, para analisar a sua consistência .

**Comment by arademaker** [comment]
Os testes seriam da conversão certo ? O objeto é garantir que a conversão não teve perdas?

**Comment by leoalenc** [comment]
@arademaker sim, isso mesmo! estou já colocando o levantamento do número de casos diferentes e o resultado da conversão.
